packages/web-components/src/components/va-radio/va-radio.tsx

```diff
@@ -82,6 +85,9 @@ export class VaRadio {
         nextNode = currentNode;
         this.selectNextNode(currentNode);
         break;
+      case 'ArrowDown':
+        // prevent scrolling page
+        event.preventDefault();
```

Comment:
This was an opportunistic fix to prevent scrolling the page when using the Down and Up arrow keys.

Reply:
Could we use fallthrough semantics to group the one below with this one, like:

```js
      case 'ArrowDown':
      case 'ArrowUp':
        // prevent scrolling page
        event.preventDefault();
      case 'ArrowDown':
      case 'ArrowRight':
        if (currentNodeIndex === radioOptionNodes.length - 1) {
          nextNode = radioOptionNodes[0];
          this.deselectCurrentNode(currentNode);
          this.selectNextNode(nextNode);
        } else {
          nextNode = radioOptionNodes[currentNodeIndex + 1];
          this.deselectCurrentNode(currentNode);
          this.selectNextNode(nextNode);
        }
        break;
```